driver-adapters/planetscale-node-basic/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "planetscale-node-basic",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "test": "jest index.test.js"
+  },
+  "devDependencies": {
+    "jest": "28.1.3",
+    "prisma": "5.3.0-integration-feat-js-connectors-in-client.13"
+  },
+  "dependencies": {
+    "@jkomyno/prisma-planetscale-js-connector": "0.0.9",
```

Comment:
The latest version is `0.0.10`, it probably doesn't matter and will be upgraded by Renovate.